community/planning/admin_application.md

```diff
@@ -0,0 +1,104 @@
+# Admin Application
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+The [splinter-ui repository](https://github.com/Cargill/splinter-ui) includes
+an example application for Splinter administration. This document shows the
+planned screens and popups for this application.
+
+
+## View Circuits and Proposals
+
+The **Circuits** screen shows this node's circuits and circuit proposals.
+
+![]({% link images/adminapp_circuits_list.png %} "Circuits list")
+
+### Circuit details
+
+After selecting a circuit ID on the **Circuits** screen, the circuit details
+screen shows information about the selected circuit.
+
+![]({% link images/adminapp_circuit_service_detail.png %} "Circuit service
+detail")
+
+## Propose a New Circuit
+
+On the **Circuits** screen, the **Propose New Circuit** button starts the
+process of defining a new circuit.
+
+![]({% link images/adminapp_button_propose_new_circuit.png %} "Propose New
+Circuit button")
+
+### Add nodes
+
+On the **Propose Circuit > Add nodes** screen, the **Next +** button
+displays the **New Node** popup.
+
+![]({% link images/adminapp_button_add_node_to_proposal.png %} "Add Node
+button")
+
+![]({% link images/adminapp_propose_popup_new_node.png %} "New Node popup")
+
+After a node has been added, the **Propose Circuit > Add nodes** screen shows
+information about the nodes in the proposed circuit.
+
+![]({% link images/adminapp_propose_circuit_two_nodes_selected.png %} "Proposed
+circuit with two nodes")
+
+### Add services
+
+![]({% link images/adminapp_propose_circuit_add_services.png %} "Add services to
+circuit proposal")
+
+### Add circuit details
+
+![]({% link images/adminapp_propose_circuit_add_circuit_details.png %} "Add
+circuit details to circuit proposal")
+
+### Add metadata
+
+![]({% link images/adminapp_propose_circuit_add_metadata.png %} "Add metadata to
+circuit proposal")
+
+### Review and submit
+
+![]({% link images/adminapp_propose_circuit_review_and_submit.png %} "Review and
+submit circuit proposal")
+
+### Confirmation
+
+Next, the admin application displays the circuit detail screen. A notification
```

Comment:
Right now, after you submit a proposal, you go back to the main screen with the circuits table. But as the disclaimer said this page does not have to be 100% accurate to what is implemented at the moment. so I don't know if this is relevant.

Reply:
Thanks! I'll fix the text and add a new image in a later PR. (The image in this PR shows the notification on the circuits detail screen.)